### PR TITLE
Page for subscribers

### PR DESCRIPTION
**Pull request by Omotola28**
Added update and add feature for subscribers list page

### CODE REVIEW

File: src/private/routes/index.js [hunk 1]
```diff
@@ -15,41 +16,42 @@ const { siteDescription } = require('../../constants/settings.js');
...
+  accountRoutes,
+  articlesRoutes,
+  candidateRoutes,
+  documentsRoutes,
+  membersRoutes,
+  pagesRoutes,
+  reviewsRoutes,
+  sessionsRoutes,
+  topicsRoutes,
+  subscribersRoutes,
```
Comment (zzavidd) [inline src/private/routes/index.js:29]: This was in alphabetic order, Aunty

File: src/constants/validations.js [hunk 2]
```diff
@@ -181,6 +181,24 @@ exports.isValidMember = (member) => {
...
+  if (!ifExists(subscriber.lastname, "Enter the subscriber's last name."))
+    return false;
+
+  const isNotSusbscribed = subscriber.subscription !== false;
+  if (!ifExists(isNotSusbscribed, 'Check box for subscribed.')) return false;
+  if (!ifExists(subscriber.createtime, "Select the subscriber's create time."))
+    return false;
+  return true;
+};
+
```
Comment (zzavidd) [inline src/constants/validations.js:201]: This isn't necessary

**Comment by zzavidd** [comment]
Also, the message that shows when there are no subscribers is wrong. It still says "No members found" instead of subscribers.

File: src/constants/validations.js (hunk 2 above)
Comment (Omotola28) [reply src/constants/validations.js:201]: This shouldnt be in this commit because it was part of the add commit

**Pull request merged**